www/data_service/src/services/dataUtils/dataUtils.service.coffee

```diff
@@ -0,0 +1,48 @@
+class DataUtils extends Service
+    constructor: ->
+        return new class DataUtils
+            constructor: ->
```

Comment:
You are right in this case, but in other occurrences I inject services 1 level up, so I don't have to bind the injected services to this (@$service).